package.json

```diff
@@ -22,13 +29,18 @@
   },
   "homepage": "https://github.com/thedeveloper/warlock",
   "dependencies": {
-    "node-redis-scripty": "~0.0.4",
-    "uuid": "2.0.1"
+    "@types/node": "17.0.10",
+    "uuid": "3.3.3"
   },
   "devDependencies": {
-    "should": "^3.1.3",
-    "mocha": "^1.17.1",
-    "redis": "^0.10.1",
-    "async": "^0.9.0"
+    "@types/mocha": "7.0.0",
+    "@types/redis": "2.8.0",
+    "async": "3.1.0",
+    "chai": "4.3.4",
+    "mocha": "9.1.4",
+    "redis": "2.8.0",
```

Comment:
Not sure if CI will complain about security vulns in devDependencies. (If it gets packaged in dist then it should.)